config/servers.yml

```diff
@@ -290,10 +290,17 @@ endpoints:
 #
 # The default value for this property is 15000 ms.
 #
-# xd:
+#xd:
 #  admin:
 #    quietPeriod: 15000
 ---
+# Module deployment timeout
+# The module deployment timeout is the amount of time the Module deployment writer
```

Comment:
What do you think about this wording?

> The module deployment timeout is the number of milliseconds the admin server will wait for a response to a module deployment request to a container. 

I think we want to avoid going too deep into the implementation details - for instance, the end user doesn't know/care about ModuleDeploymentWriter, ZK, or paths.